modules/mcagar/src/main/ts/ephox/mcagar/alien/Settings.ts

```diff
@@ -0,0 +1,31 @@
+import { Editor } from './EditorTypes';
+
+const getSetting = <R>(editor: Editor, name: string): R => {
```

Comment:
Maybe it will be just confusing to any future developer that these functions are called `getSetting` since the options name is the way forward.

Reply:
Have renamed this and deprecated the old functions